tests/Model/Svg/SvgFileTest.php

```diff
@@ -476,6 +476,13 @@ public function testSwitchTranslationSetRoundtrip(): void
         $this->assertEquals(str_replace(' ', '', $origXml), str_replace(' ', '', $this->svg->saveToString()));
     }
 
+    public function testSetTranslations(): void
+    {
+        // The test file does not contain Spanish.
+        $this->svg->setTranslations('es', ['tspan2993' => 'FooBar']);
+        static::assertContains('FooBar', $this->svg->saveToString());
```

Comment:
So I'm a bit confused about this test, I think it's not completely clear. We set a single translation with a language, and when we ask for the file, we see that translation... I am still a little confused with the use of plural-translations but that's an aside. I think what would help here (and make the test better) is if you setTranslation() more than once and then verify what you get. I'm a little confused if we're testing the preview functionality (so you only get the latest set language?) or if we're testing the general functionality (so you get all available translations) but either way, setting it twice will resolve that confusion by merely what the test expects.

For example, if we test that only the latest was added (for only the preview case, not caring about previously set languages)
```
 $this->svg->setTranslations('es', ['tspan2993' => 'FooBar']);
 $this->svg->setTranslations('fr', ['tspan2993' => 'BarFoo']);
// Test only the latest counts
static::assertContains('BarFoo', $this->svg->saveToString());
static::assertNotContains('FooBar', $this->svg->saveToString(BarFoo))
```

If `saveToString` does take into account all languages then you can just make sure both of those check on `assertContains`

Reply:
Multiple languages can be set, but we don't use that functionality. I've added tests to make it a bit clearer.